fossunited/fossunited/doctype/grants_funding_directory/grants_funding_directory.py

```
@@ -0,0 +1,259 @@
+            json_content = self._fetch_json_from_url(self.funding_json)
+
+            # Validate using official API
+            validated_data = self._validate_json_with_api(self.funding_json, json_content)
+
+            # Store validated JSON
+            self.json_data = json.dumps(validated_data, indent=2)
+            self.last_updated = now_datetime()
+
+        except requests.exceptions.RequestException as e:
+            frappe.log_error(
+                f"Error fetching funding JSON for {self.name or 'new'}: {str(e)}",
```

Ruff (0.14.5)

Use explicit conversion flag

Replace with conversion flag

(RUF010)